tests/unit/test_config.py

```
@@ -0,0 +1,64 @@
+        "ADO_PROJECT",
+        "ADO_PAT",
+        "ADO_SERVICE_HOOK_SECRET",
+        "DATABRICKS_HOST",
+        "DATABRICKS_TOKEN",
+    ):
+        monkeypatch.delenv(name, raising=False)
+
+    sys.modules.pop("config", None)
+    config = importlib.import_module("config")
+
+    with pytest.raises(Exception):
```

Ruff (0.15.9)

[warning] Do not assert blind exception: `Exception`

(B017)